packages/manager/src/features/CloudPulse/shared/CloudPulseDashboardSelect.tsx

```diff
@@ -90,11 +94,8 @@ export const CloudPulseDashboardSelect = React.memo(
         }}
         renderGroup={(params) => (
           <Box key={params.key}>
-            <Typography
-              sx={{ marginLeft: '3.5%', textTransform: 'capitalize' }}
-              variant="h3"
-            >
-              {params.group}
+            <Typography sx={{ marginLeft: '3.5%' }} variant="h3">
+              {serviceTypeMap.get(params.group)}
```

Comment:
serviceTypeMap.get(params.group), if the map doesn't have the key, lets display the service_type and not show undefined